src/Visualization/Python/Render1D.py

```diff
@@ -0,0 +1,279 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+# Python script to render 1D data written using SpECTRE in format v2
+
+import glob
+import h5py
+import argparse
+import sys
+import os
+import numpy as np
+import logging
+import matplotlib as mpl
+try:
+    import Tkinter as tkinter
+except ImportError:
+    import tkinter
+
+
+def find_extrema_over_data_set(arr):
+    '''
+    Find max and min over a range of number arrays
+    :param arr: the array over which to find the max and min
+    '''
+
+    return (np.nanmin(arr), np.nanmax(arr))
+
+
+def parse_cmd_line():
+    '''
+    parse command-line arguments
+    :return: dictionary of the command-line args, dashes are underscores
+    '''
+
+    parser = argparse.ArgumentParser(
+        description='Render 1-dimensional data',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    group_files = parser.add_mutually_exclusive_group(required=True)
+    group_files.add_argument(
+        '--file-prefix',
+        type=str,
+        help='Common prefix of all h5 files being used '
+        'in the case of a single h5 file, it is the file name '
+        'without the .h5 extension')
+    group_files.add_argument('--filename-list',
+                             type=str,
+                             nargs='+',
+                             help='List of all files if they do not '
+                             'have same file prefix. You must include \'.h5\' '
```

Comment:
Just an FYI (not a change request): you can use different quotation mark styles to avoid the escaping with `\`. E.g. if you use `"` for the string then you can use `'` within it.